CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Security Baseline
+
+## Contributing to the Criteria:
+
+Currently, all the baseline criteria are in `baseline.yaml`; follow the structure
+[in the README](./README.md#baseline-structure) when proposing new entries.
+
+## Contributing to the Tooling:
+
+The baseline is published to https://baseline.openssf.org/ (via GitHub Pages) via
+Jekyll (a static site generator) using scripts from `./cmd` and formatting from
+`./docs`, using GitHub Actions.
+
+## PR guidelines
+
+All changes to the repository should be made via PR
+([OSPS-AC-03](https://baseline.openssf.org/#osps-ac-03)).  In addition to a clear
+title and descriptive commit message, PRs MUST meet the following criteria:
+
+* DCO signoff (via `git commit -s` -- [OSPS-LE-01](https://baseline.openssf.org/#osps-le-01))
+* All checks must pass ([OSPS-QA-04](https://baseline.openssf.org/#osps-qa-04))
```

Comment:
We might want to leave the links out of this, since #113 will break them anyway. I think we're fine without the citations, but if you feel strongly about leaving them in, we should decide if we want to have them be hyperlinks or just bare text references.

Reply:
As "customer zero", we should figure out what the best practices are here.  Linking to the criteria seems helpful in providing background information, but only if the URLs are stable enough that they don't just turn into dead links.

(And, I think that's something worth thinking about as we proceed...)